<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Encrypt_settings_window</class>
 <widget class="QDialog" name="Encrypt_settings_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPlainTextEdit" name="plainText1Edit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>391</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_Text1">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>4</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Text1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Key1</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="text1Key1Edit">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>60</y>
     <width>391</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_Text2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Text2</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainText2Edit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>371</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="text1Key2Edit">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>300</y>
     <width>381</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>250</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Key2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonEncryptFinally">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>550</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Encrypt!</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
